packages/graphback-core/src/plugin/getSelectedFieldsFromResolverInfo.ts

```diff
@@ -21,6 +21,8 @@ export const getSelectedFieldsFromResolverInfo = (info: GraphQLResolveInfo, mode
 
         if (foundRelationship) {
           acc.push(foundRelationship.relationForeignKey)
+        } else {
```

Comment:
We'll need to check if it is not a `oneToMany` relationship before adding it.

Without it a relationship like this one below 
```
""" @model """
type Note {
  id: ID!
  title: String!
  description: String
  """
  @oneToMany(field: 'note')
  """
  comments: [Comment]!
}

""" @model """
type Comment {
  id: ID!
  text: String
  description: String
}
```

Would fail when issue a graphql request of this sort
```
findNotes {

 items {
  id, 
  comments {
    id
  }
}
}

```

it will fail consider `comments` as selectable field while it should not.

Reply:
Pushed a https://github.com/aerogear/graphback/commit/34365254d29a4146231169e1d409f998ab11968d to fix this issue and make postgres integration tests to pass. Working on the Mongo test cases now.